src/YieldDisburser.sol

```diff
@@ -55,17 +56,24 @@ contract YieldDisburser is OwnableUpgradeable {
         breadToken.claimYield(breadToken.yieldAccrued(), address(this));
 
         (uint256[] memory projectDistributions, uint256 totalVotes) = _getVotedDistribution(breadchainProjects.length);
-
         lastClaimedTimestamp = Time.timestamp();
         lastClaimedBlocknumber = Time.blockNumber();
 
         uint256 halfBalance = breadToken.balanceOf(address(this)) / 2;
         uint256 baseSplit = halfBalance / breadchainProjects.length;
+        uint256 percentage_of_total_vote;
+        uint256 votedSplit;
+        uint256[] memory votedSplits;
+        uint256[] memory percentages;
 
         for (uint256 i; i < breadchainProjects.length; ++i) {
-            uint256 votedSplit = ((projectDistributions[i] * halfBalance) / totalVotes);
+            percentage_of_total_vote = projectDistributions[i] / totalVotes;
```

Comment:
is this not the same problem with integer division result in 0 every time because `projectDistributions[i]` will always be less than `totalVotes`?

Reply:
This pr was created before the percision, which is in a different PR. If we can merge this so I will amend this in the other PR, I just don't want to merge this pr to the other one. Leaving a comment there.